org.telegram.desktop.json

```diff
@@ -98,91 +87,173 @@
                 }
             ]
         },
+        {
+            "name": "cmake",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Kitware/CMake/releases/download/v3.16.2/cmake-3.16.2.tar.gz",
+                    "sha256": "8c09786ec60ca2be354c29829072c38113de9184f29928eb9da8446a5f2ce6a9"
+                }
+            ],
+            "cleanup": [ "*" ]
+        },
+        {
+            "name": "enchant",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/AbiWord/enchant/releases/download/v2.2.7/enchant-2.2.7.tar.gz",
+                    "sha256": "1b22976135812b35cb5b8d21a53ad11d5e7c1426c93f51e7a314a2a42cab3a09"
+                }
+            ]
+        },
+        {
+            "name": "lz4",
```

Comment:
lz4 seems to also be in the runtime

Reply:
Nope, no lz4 in the runtime.

Reply:
Ok, you seem to be right. I was checking source code of the freedesktop runtime and saw it so I assumed it's there.